fe/fe-core/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -758,6 +764,22 @@ public static void checkAndCreateSource(Database db, DataDescription dataDescrip
                         // do nothing
                     }
 
+                } else if (!column.isVisible()) {
+                    /*
+                     *  For batch delete table add hidden column __DORIS_DELETE_SIGN__ to columns
+                     * eg:
+                     * (A, B, C)
+                     * ->
+                     * (A, B, C) SET (__DORIS_DELETE_SIGN__ = 0)
+                     */
+                    columnToHadoopFunction.put(column.getName(), Pair.create("default_value", Lists.newArrayList("0")));
+                    ImportColumnDesc importColumnDesc = null;
+                    try {
+                        importColumnDesc = new ImportColumnDesc(column.getName(), new FunctionCallExpr("default_value", Arrays.asList(column.getDefaultValueExpr())));
```

Comment:
I think it's better to extract `default_value ` to a common constant to decrease hard code

Reply:
`default_value` is not change for a long time, and hadoop load is deprecated, this pr just for  compitable for old systems